plugins/forwarder/kafka/nativemanagement/forwarder.go

```diff
@@ -0,0 +1,110 @@
+// Licensed to Apache Software Foundation (ASF) under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Apache Software Foundation (ASF) licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package nativemanagement
+
+import (
+	"fmt"
+	"reflect"
+
+	"github.com/Shopify/sarama"
+
+	"github.com/apache/skywalking-satellite/internal/pkg/config"
+	"github.com/apache/skywalking-satellite/internal/satellite/event"
+	"google.golang.org/protobuf/proto"
+	management "skywalking.apache.org/repo/goapi/collect/management/v3"
+
+	v1 "skywalking.apache.org/repo/goapi/satellite/data/v1"
+)
+
+const (
+	Name     = "nativemanagement-kafka-forwarder"
+	ShowName = "Native Management Kafka Forwarder"
+)
+
+// empty struct for set
+type void struct{}
+
+var empty void
```

Comment:
I think these fields and struct are not used, why do you declare in there?

Reply:
> Thanks for your contribution, please complete the following steps:
> 
> 1. Fix the comments.
> 2. Using `make gen-docs` to generate the documentation.
> 3. You need to create an E2E to verify the Kafka forwards are working well.

Sure will check and fix all the comments added..